docs/guide/search.md

```diff
@@ -299,9 +299,27 @@ $searchResult = $searchService->findContent( $query );
 $content = $searchResult->searchHits[0];
 ```
 
-### Reindexing
+## Reindexing
 
-To (re)create the search engine index for configured search engines (per siteaccess repository), use the `php app/console ezplatform:reindex` command.
+To (re)create or refresh the search engine index for configured search engines (per siteaccess repository), use the `php app/console ezplatform:reindex` command.
+
+Some examples of common usage:
+```bash
+# Reindex whole index using parallel process
```

Comment:
To make the sentences below a bit more clear in regards to purging:
`# Reindex whole index using parallel process (by default starts by purging whole index)`